funcions/4.js

```diff
@@ -0,0 +1,31 @@
+function oldRemoveDuplicates(array) {
+	var newArray = []
+	var valIsPresent = false
+	for (var i = 0; i < array.length; i++) {
+		for (var x = 0; x < newArray.length; x++) {
+			if (newArray[x] === array[i]) {
+				valIsPresent = true
+			}
+		}
+		valIsPresent ? null : newArray.push(array[i])
+
+		valIsPresent = false
+	}
+	return newArray
+}
+
+function newRemoveDuplicates(array) {
+	const newArray = []
+	array.forEach((item) => {
+		if (!newArray.includes(item)) {
+			newArray.push(item)
+		}
+	})
```

Comment:
Looks like a classic situation where `.reduce` could be used :)

Reply:
What  would be the benefit of using reduce here?

Reply:
You wouldn't need to create a temporary array to hold your result. Basically, just a shorter way to write the same logic down.  
Also consider looking at a `Set` if you haven't used it yet. 😉 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set